fiftyone/core/models.py

```diff
@@ -157,7 +157,7 @@ def apply_model(
         )
 
     if classes is not None:
-        classes = set(classes)
+        classes = list(set(classes))
```

Comment:
I had intentionally converted to `set()` here so that `label in classes` lookups will be `O(1)` instead of `O(len(classes))`.